### PR TITLE
Enhancement/82 improve documentation level

### PR DESCRIPTION
**Pull request by frodoslaw**
Improve documentation level by writing Rdocs #82

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/coi-gov-pl/puppet-jboss/pull/83" target="_blank">ReviewNinja

File: lib/puppet_x/coi/jboss/provider/datasource.rb [hunk 1]
```diff
@@ -57,10 +59,7 @@ def prepare_resource
     if @resource[:name].nil?
       @resource[:name] = @property_hash[:name]
     end
-    if @resource[:controller].nil?
-      @resource[:controller] = controller
-    end
-    if @resource[:runasdomain].nil?
+    if @resource[:controller].nilzabbix sent two messagesl?
```
Comment (Cosaquee) [inline lib/puppet_x/coi/jboss/provider/datasource.rb]: 👍

**Comment by cardil** [comment]
:+1:

**Comment by Cosaquee** [comment]
LGFM

**Comment by cardil** [comment]
Lgtm 👍

**Pull request merged**